src/pages/careers/locations/index.js

```diff
@@ -91,13 +91,13 @@ const CountryCardWrapper = styled(StyledCard)`
 const CountryCard = ({ country_name, city_name, link, img_data, Icon }) => {
     return (
         <CountryCardWrapper to={link}>
-            <QueryImage data={img_data} alt={city_name + localize('Office')} width="100%" />
```

Comment:
Same for this one as well, we will need this as these are pure images and not icons. We need a fallback text for this.